src/overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -145,22 +170,46 @@ def display_states(self, *states):
             print(self)
         self.state = old_state
 
-    def print_state_transition(self, a_t, r_t, env_info):
+    def print_state_transition(self, a_t, r_t, env_info, fname=None):
         """
         Terminal graphics visualization of a state transition.
         """
         # TODO: turn this into a "formatting action probs" function and add action symbols too
         action_probs = [None if "action_probs" not in agent_info.keys() else list(agent_info["action_probs"]) for agent_info in env_info["agent_infos"]]
-        action_probs = [ None if player_action_probs is None else [round(p, 2) for p in player_action_probs] for player_action_probs in action_probs ]
-        print("Timestep: {}\nJoint action taken: {} \t Reward: {} + shaping_factor * {}\nAction probs by index: {}\nState potential = {}\n{}\n".format(
-                self.state.timestep,
-                tuple(Action.ACTION_TO_CHAR[a] for a in a_t),
-                r_t,
-                env_info["shaped_r_by_agent"],
-                action_probs,
-                self
+
+        action_probs = [ None if player_action_probs is None else [round(p, 2) for p in player_action_probs[0]] for player_action_probs in action_probs ]
+
+        if fname is None:
+            print("Timestep: {}\nJoint action taken: {} \t Reward: {} + shaping_factor * {}\nAction probs by index: {}\nState potential = {} \t Δ potential = {} \n{}\n".format(
+                    self.state.timestep,
+                    tuple(Action.ACTION_TO_CHAR[a] for a in a_t),
+                    r_t,
+                    env_info["shaped_r_by_agent"],
+                    action_probs,
+                    "",
+                    # self.mdp.potential_function(self.state, self.mlp.mp),
+                    "",
+                    # 0.99 * env_info["phi_s_prime"] - env_info["phi_s"], # Assuming gamma 0.99
```

Comment:
This is slightly more problematic. I moved this to the channel to ask for Nathan's confirmation. I think the problem here is that we don't have a default onion_value or tomato_value, since they are pretty layout-specific.  but without those information, we can't use the most up-to-date potential function. @nathan-miller23 Please chime in on this issue. Thanks!

Reply:
I already have a phi_s fix in the pipeline, but that won't work unless the potential function is fixed

Reply:
I see, sounds good!

Reply:
After talking to Nathan, this has been fixed in the newest commit